src/extensions/nginx-app-protect/nap/attack_signatures.go

```diff
@@ -0,0 +1,47 @@
+package nap
```

Comment:
Hi, Can you add the copyright header to all of the new files. Thank you!

Reply:
Done!